recipes/qsymm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qsymm" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bc144ed6ac3c8033e174e8c1dba788876eaf6ccc98bca727b4b325e35e572973
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
```

Comment:
You probably only need `python` and `pip` as host requirements.

Reply:
Thanks!